server/main.lua

```diff
@@ -143,10 +143,10 @@ end)
 local function sendDoctorAlert()
 	if doctorCalled then return end
 	doctorCalled = true
-	local _, doctors = QBCore.Functions.GetDutyCountJob('ambulance')
+	local _, doctors = QBCore.Functions.GetDutyCountType('ems')
 	for i = 1, #doctors do
 		local doctor = doctors[i]
```

Comment:
@Manason why is this localized instead of directly pointing at it in the next line?